### ISSUE TITLE
Analyze tab is no longer visiable by default in master

### ISSUE THREAD

**Issue by yyjdelete** [icsharpcode/ILSpy#1803]
Tested with master, seems broken after #1801 .

Maybe an clean `%appdata%\ICSharpCode\ILSpy.xml` is needed to reproduce.
1. Remove `%appdata%\ICSharpCode\ILSpy.xml`;
2. Open master builds of ILSpy, Right click any classes, and click `Analyze`;
> And see no tab is shown;
3. Close and reopen ILSpy again;
>See an blank zone.
![image](https://user-images.githubusercontent.com/1819074/69206017-8b5ef200-0b86-11ea-9186-4f4ca387fc08.png)
4. Click `Analyze` again;
>see it display in blank zone.
![image](https://user-images.githubusercontent.com/1819074/69206039-974ab400-0b86-11ea-96a2-cc3759ddaf4a.png)

And the same thing also happen with `show debug steps` in debug builds.

BTW: 
Not sure if it's the same issue. After you run the master builds, without reset `ILSpy.xml` or open ILSpy5.x, you can see an empty UI when you run 6.x builds before #1801.
![image](https://user-images.githubusercontent.com/1819074/69206716-61a6ca80-0b88-11ea-8bfd-3e20e6b87d8c.png)

Edited:
Also for `search` tab.

**Comment by siegfriedpammer** [comment]
Thank you for reporting this!

Can you please try the latest master?

**Comment by yyjdelete** [comment]
@siegfriedpammer 

The original issue is fixed, but seems another variant.
1. Right click and click `Float` on the main(code) tab of "Resource"(or just drag the tab)
```cs
System.ArgumentException: "-44" is not an valid value for property "Width".
   at System.Windows.DependencyObject.SetValueCommon(DependencyProperty dp, Object value, PropertyMetadata metadata, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType, Boolean isInternal)
   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Object value)
   at ICSharpCode.ILSpy.Controls.ResourceStringTable.OnParentSizeChanged(Object sender, SizeChangedEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.FrameworkElement.OnRenderSizeChanged(SizeChangedInfo sizeInfo)
   at System.Windows.ContextLayoutManager.fireSizeChangedEvents()
   at System.Windows.ContextLayoutManager.UpdateLayout()
   at System.Windows.Interop.HwndSource.SetLayoutSize()
   at System.Windows.Interop.HwndSource.set_RootVisualInternal(Visual value)
   at System.Windows.Window.SetRootVisual()
   at System.Windows.Window.SetRootVisualAndUpdateSTC()
   at System.Windows.Window.SetupInitialState(Double requestedTop, Double requestedLeft, Double requestedWidth, Double requestedHeight)
   at System.Windows.Window.CreateSourceWindow(Boolean duringShow)
   at System.Windows.Window.ShowHelper(Object booleanBox)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```
-----
And another 2 issues for new UI with tabs.

2. Right click and click `Decompile to new tab` in `Analyze` tab;
```cs
System.InvalidCastException: Objects of type "ICSharpCode.ILSpy.Analyzers.TreeNodes.AnalyzedTypeTreeNode" cannot be cast to type "ICSharpCode.ILSpy.TreeNodes.ILSpyTreeNode".
   at System.Linq.Enumerable.<CastIterator>d__97`1.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at ICSharpCode.ILSpy.Commands.DecompileInNewViewCommand.Execute(TextViewContext context)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.Controls.MenuItem.InvokeClickAfterRender(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

----
3. I'm not sure the steps, but one possible way to reproduce:
3.1. Open the search tab;
3.2. Drag and drop the tab to bottom;

![image](https://user-images.githubusercontent.com/1819074/69247680-bf1d3480-0be5-11ea-919a-78bb0ff50634.png)

3.3. Close the search tab;
3.4. Open the search tab again, and the UI looks strange now;
![image](https://user-images.githubusercontent.com/1819074/69248057-829e0880-0be6-11ea-8d48-2cb3cb143681.png)


3.5. Drag the search tab and see the NRE;
```cs
System.NullReferenceException: 
   at Xceed.Wpf.AvalonDock.Layout.LayoutContent.CloseInternal()
   at Xceed.Wpf.AvalonDock.Layout.LayoutAnchorable.CloseAnchorable()
   at Xceed.Wpf.AvalonDock.Controls.LayoutItem.OnVisibilityChanged()
   at Xceed.Wpf.AvalonDock.Controls.LayoutItem.<>c.<.cctor>b__13_1(DependencyObject s, DependencyPropertyChangedEventArgs e)
   at System.Windows.PropertyChangedCallback.Invoke(DependencyObject d, DependencyPropertyChangedEventArgs e)
   at System.Windows.DependencyObject.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
   at System.Windows.FrameworkElement.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
   at System.Windows.DependencyObject.NotifyPropertyChange(DependencyPropertyChangedEventArgs args)
   at System.Windows.DependencyObject.UpdateEffectiveValue(EntryIndex entryIndex, DependencyProperty dp, PropertyMetadata metadata, EffectiveValueEntry oldEntry, EffectiveValueEntry& newEntry, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType)
   at System.Windows.DependencyObject.SetValueCommon(DependencyProperty dp, Object value, PropertyMetadata metadata, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType, Boolean isInternal)
   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Object value)
   at Xceed.Wpf.AvalonDock.Controls.LayoutAnchorableItem.SetDefaultBindings()
   at Xceed.Wpf.AvalonDock.DockingManager.ApplyStyleToLayoutItem(LayoutItem layoutItem)
   at Xceed.Wpf.AvalonDock.DockingManager.CreateAnchorableLayoutItem(LayoutAnchorable contentToAttach)
   at Xceed.Wpf.AvalonDock.DockingManager.Layout_ElementAdded(Object sender, LayoutElementEventArgs e)
   at Xceed.Wpf.AvalonDock.Layout.LayoutRoot.OnLayoutElementAdded(LayoutElement element)
   at Xceed.Wpf.AvalonDock.Layout.LayoutElement.OnRootChanged(ILayoutRoot oldRoot, ILayoutRoot newRoot)
   at Xceed.Wpf.AvalonDock.Layout.LayoutElement.set_Parent(ILayoutContainer value)
   at Xceed.Wpf.AvalonDock.Layout.LayoutRoot._floatingWindows_CollectionChanged(Object sender, NotifyCollectionChangedEventArgs e)
   at System.Collections.ObjectModel.ObservableCollection`1.OnCollectionChanged(NotifyCollectionChangedEventArgs e)
   at System.Collections.ObjectModel.ObservableCollection`1.InsertItem(Int32 index, T item)
   at Xceed.Wpf.AvalonDock.DockingManager.CreateFloatingWindowForLayoutAnchorableWithoutParent(LayoutAnchorablePane paneModel, Boolean isContentImmutable)
   at Xceed.Wpf.AvalonDock.DockingManager.StartDraggingFloatingWindowForPane(LayoutAnchorablePane paneModel)
   at Xceed.Wpf.AvalonDock.Controls.AnchorablePaneTitle.OnMouseLeave(MouseEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.FirePropertyChangeInAncestry(DependencyObject element, Boolean oldValue, DeferredElementTreeState treeState, Action`2 originChangedAction)
   at System.Windows.ReverseInheritProperty.OnOriginValueChanged(DependencyObject oldOrigin, DependencyObject newOrigin, IList`1 otherOrigins, DeferredElementTreeState& oldTreeState, Action`2 originChangedAction)
   at System.Windows.Input.MouseDevice.ChangeMouseOver(IInputElement mouseOver, Int32 timestamp)
   at System.Windows.Input.MouseDevice.PreNotifyInput(Object sender, NotifyInputEventArgs e)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by siegfriedpammer** [comment]
@yyjdelete I think, I have fixed all issues you found. Could you please test again? Thank you very much!

If you find anything else, please open a new issue. Thank you!

**Comment by yyjdelete** [comment]
@siegfriedpammer All fixed. 👍
